dotCMS/src/main/java/com/dotcms/api/system/event/SystemEventType.java

```diff
@@ -29,6 +29,8 @@
 
 	SAVE_BASE_CONTENT_TYPE,
 	UPDATE_BASE_CONTENT_TYPE,
-	DELETE_BASE_CONTENT_TYPE;
+	DELETE_BASE_CONTENT_TYPE,
```

Comment:
why delete the DELETE_BASE_CONTENT_TYPE event type?, I think that it is use in somewhere